media-libs/opencolorio/opencolorio-2.3.2.ebuild

```diff
@@ -0,0 +1,173 @@
+			dev-python/sphinx[${PYTHON_USEDEP}]
+			dev-python/sphinx-press-theme[${PYTHON_USEDEP}]
+			dev-python/sphinx-tabs[${PYTHON_USEDEP}]
+			dev-python/testresources[${PYTHON_USEDEP}]
+		')
+	)
+	opengl? (
+		media-libs/freeglut
+		media-libs/glew:=
+		media-libs/libglvnd
+	)
+"
+# 	test? (
+# 		>=media-libs/openimageio-2.2.14
+# 		>=media-libs/osl-1.11
+# 	)
+# "
+
+# Restricting tests, bugs #439790 and #447908
+# upstream doesn't care https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1361
+RESTRICT="test" # "!test? ( test )"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-2.2.1-adjust-python-installation.patch"
+)
+
+pkg_setup() {
+	use python && python-single-r1_pkg_setup
+}
+
+src_prepare() {
+	cmake_src_prepare
+
+	# Avoid automagic test dependency on OSL, bug #833933
+	# Can cause problems during e.g. OpenEXR unsplitting migration
+	cmake_run_in tests cmake_comment_add_subdirectory osl
+}
+
+src_configure() {
+	# Missing features:
+	# - Truelight and Nuke are not in portage for now, so their support are disabled
+	# - Java bindings was not tested, so disabled
+	# Notes:
+	# - OpenImageIO or OpenEXR (default) is required for building ociodisplay and
+	#	ocioconvert (USE opengl)
+	# - OpenGL, GLUT and GLEW is required for building ociodisplay (USE opengl)
+	local mycmakeargs=(
+		"-DOCIO_BUILD_APPS=$(usex apps)"
+		"-DOCIO_BUILD_DOCS=$(usex doc)"
+		"-DOCIO_BUILD_FROZEN_DOCS=$(usex doc)"
+		"-DOCIO_BUILD_GPU_TESTS=$(usex test)"
+		"-DOCIO_BUILD_JAVA=OFF"
+		"-DOCIO_BUILD_PYTHON=$(usex python)"
+		"-DOCIO_BUILD_TESTS=$(usex test)"
+		"-DOCIO_INSTALL_EXT_PACKAGES=NONE"
+		# allow the user to tell OCIO to display more information when searching and building the dependencies.
+		# "-DOCIO_VERBOSE=YES"
+
+		"-DOCIO_USE_SIMD=ON"
+	)
+
+	if use amd64 || use x86 ; then
+		mycmakeargs+=(
+			"-DOCIO_USE_SSE2=$(usex cpu_flags_x86_sse2)"
+			"-DOCIO_USE_SSE3=$(usex cpu_flags_x86_sse3)"
+			"-DOCIO_USE_SSSE3=$(usex cpu_flags_x86_ssse3)"
+			"-DOCIO_USE_SSE4=$(usex cpu_flags_x86_sse4_1)"
+			"-DOCIO_USE_SSE42=$(usex cpu_flags_x86_sse4_2)"
+			"-DOCIO_USE_AVX=$(usex cpu_flags_x86_avx)"
+			"-DOCIO_USE_AVX2=$(usex cpu_flags_x86_avx2)"
+			"-DOCIO_USE_AVX512=$(usex cpu_flags_x86_avx512f)"
+			"-DOCIO_USE_F16C=$(usex cpu_flags_x86_f16c)"
+		)
+	fi
+
+	# requires https://github.com/DLTcollab/sse2neon
+	# if use arm || use arm64 ; then
+	# 	mycmakeargs+=(
+	# 		"-DOCIO_USE_SSE2NEON=$(usex cpu_flags_arm_neon)"
+	# 	)
+	# fi
+
+	use python && mycmakeargs+=(
+		"-DOCIO_PYTHON_VERSION=${EPYTHON/python/}"
+		"-DPython_EXECUTABLE=${PYTHON}"
+		"-DPYTHON_VARIANT_PATH=$(python_get_sitedir)"
+	)
+
+	cmake_src_configure
+}
+
+src_install() {
+	cmake_src_install
+
+	if use doc; then
+		# there are already files in ${ED}/usr/share/doc/${PF}
+		mv "${ED}/usr/share/doc/OpenColorIO/"* "${ED}/usr/share/doc/${PF}" || die
+		rmdir "${ED}/usr/share/doc/OpenColorIO" || die
+	fi
+}
+
+src_test() {
+	local myctestargs=(
+		-j1
+	)
+	nonfatal \
+	virtx cmake_src_test
```

Comment:
nonfatal with no die...?

Reply:
`nonfatal` has no consequences with neither `virtx` nor `cmake_src_test` because they don't pass `die -n` for the main `die` or anywhere else.
See [cmake](https://github.com/gentoo/gentoo/blob/master/eclass/cmake.eclass#L706-L708) [virtualx](https://github.com/gentoo/gentoo/blob/master/eclass/virtualx.eclass#L143).

This was there to allow the tests to run without restricting them.